### PR TITLE
Added a new resource

### PR DESCRIPTION
**Pull request by sandeepmishratech**
Added a new resource under 'C'. This C Programming Tutorial is explained with real-world examples and provides hands-on experience for various coding problems. Hope this small contribution will help the learners.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,6 +179,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - :books: [Learn C Org](https://www.learn-c.org)
 - :bulb: [C Cheatsheet](https://www.codewithharry.com/blogpost/c-cheatsheet)
+- :file_folder: [C Programming Tutorial](https://www.scaler.com/topics/c/)
```
Comment (shahednasser) [inline README.md:182]: Change the emoji to `:books:` and change the title to the same title as the page.

**Comment by sandeepmishratech** [comment]
hi @shahednasser,

I have made all the changes, please check.

Thank You!

File: README.md [hunk 2]
```diff
@@ -179,6 +179,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - :books: [Learn C Org](https://www.learn-c.org)
 - :bulb: [C Cheatsheet](https://www.codewithharry.com/blogpost/c-cheatsheet)
+- :books: [C Tutorial](https://www.scaler.com/topics/c/)
```
Comment (shahednasser) [inline README.md:182]: The title is longer than this as I can see when I open the page. Please set it to "C Tutorial - Learn C in Detail" and do the same for the other pages.

**Comment by sandeepmishratech** [comment]
Hi @shahednasser,

I have made all the changes as suggested, please check.

Thank You!

File: README.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - :books: [Learn C Org](https://www.learn-c.org)
 - :bulb: [C Cheatsheet](https://www.codewithharry.com/blogpost/c-cheatsheet)
-- :books: [C Tutorial](https://www.scaler.com/topics/c/)
+- :books: [C Tutorial | Learn C in Detail](https://www.scaler.com/topics/c/)
```
Comment (shahednasser) [inline README.md:182]: change `|` to `-` and please do it everywhere you've used it.

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by sandeepmishratech** [comment]
Hi @shahednasser,

I have made the changes.

**Comment by shahednasser** [comment]
Thanks!

**Pull request merged**